charts/datadog/values.yaml

```diff
@@ -283,6 +283,10 @@ datadog:
     ## ref: TODO - add doc link
     enabled: false
 
+  networkConfig:
```

Comment:
I'm wondering if the "namespace" should be more align with the product name `networkPerformanceMonitoring` or just "npm". 

I feel like with `networkConfig` people can misunderstood the purpose of this section.

Reply:
I agree that `networkConfig` could be misleading.   I chose  `networkConfig` for parity with the `nework_config` section in the system-probe.yaml file. That's already in 7.23.0 so we can't easily change it.

I'm not sure how much parity we should keep with the system-probe.yaml formatting in the helm options, which I could use feedback on. I'd avoid `npm` though, since that acronym usually refers to "node package manager"

Reply:
I think we don't need feature parity between the system-probe.yaml config file and the option in the helm chart. 
I know that in the past we weren't very "clean" about naming convention, but we should try to use  name that "talk" to the user.